src/services/subject.js

```
@@ -32,8 +32,8 @@ const subjectService = {
   deleteSubject: async (id) => {
     await Subject.findByIdAndRemove(id).exec()
   },
-  getSubjectsByCategoryId: async (categoryId) => {
-    return Subject.find({ category: ObjectId(categoryId) })
+  getSubjectNamesByCategoryId: async (categoryId) => {
+    return Subject.find({ category: ObjectId(categoryId) }, { name: 1 })
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)